### PR TITLE
Adding more small fixes to the walkthrough & install docs

### PR DESCRIPTION
**Pull request by arschles**
This is a follow-up to #1163.

### CODE REVIEW

File: docs/walkthrough-1.6.md [hunk 1]
```diff
@@ -277,9 +277,9 @@ ups-instance-credential           Opaque                                2
 
 Notice that a new `Secret` named `ups-instance-credential` has been created.
 
-# Step 6 - Unbinding from the ServiceInstance
+# Step 6 - Deleting the `ServiceInstanceCredentials`
```
Comment (pmorie) [inline docs/walkthrough-1.6.md]: nit, this is singular instead of plural

**Comment by pmorie** [comment]
One nit and LGTM

File: docs/walkthrough-1.6.md (hunk 1 above)
Comment (arschles) [reply docs/walkthrough-1.6.md]: good catch, thanks

Comment (arschles) [reply docs/walkthrough-1.6.md]: done

**Comment by arschles** [comment]
Thanks for the review @pmorie

File: docs/walkthrough-1.6.md [hunk 2]
```diff
@@ -11,11 +11,11 @@ deprecated and may be removed at any time.
...
 see [walkthrough-1.7.md](./walkthrough-1.7.md).
 
-This document assumes that you've installed Service Catalog onto your cluster.
+__Note: This document assumes that you've installed Service Catalog onto your cluster.
 If you haven't, please see the 
-[installation instructions for 1.6](./install-1.6.md).
+[installation instructions for 1.6](./install-1.6.md).__
 
-# Step 1 - Installing the UPS ServiceBroker
+# Step 1 - Installing the UPS Broker Server
```
Comment (duglin) [inline docs/walkthrough-1.6.md]: Hmmm, isn't "Service Broker" more appropriate?

File: docs/walkthrough-1.6.md [hunk 3]
```diff
@@ -277,9 +277,9 @@ ups-instance-credential           Opaque                                2
 
 Notice that a new `Secret` named `ups-instance-credential` has been created.
 
-# Step 6 - Unbinding from the ServiceInstance
+# Step 6 - Deleting the `ServiceInstanceCredential`
 
-Now, let's unbind from the instance.  To do this, we simply *delete* the
+Now, let's unbind from the provisioned instance. To do this, we simply *delete* the
```
Comment (duglin) [inline docs/walkthrough-1.6.md]: seems inconsistent to use 'delete' in the title but 'unbind' in the text.  I'm ok with either one, but we should be consistent

File: docs/walkthrough-1.6.md (hunk 2 above)
Comment (duglin) [reply docs/walkthrough-1.6.md]: or even "Service Broker Server" if you're trying to be less OSB-API-y

File: docs/walkthrough-1.6.md (hunk 3 above)
Comment (arschles) [reply docs/walkthrough-1.6.md]: @duglin this document tries to equate Kubernetes actions (delete in this case) with the equivalent OSB concept (unbind in this case), so I kept with that trend.

File: docs/walkthrough-1.6.md (hunk 2 above)
Comment (arschles) [reply docs/walkthrough-1.6.md]: I can do "Service Broker Server". I need to clearly differentiate it from the `ServiceBroker` resource.

Comment (arschles) [reply docs/walkthrough-1.6.md]: done

File: docs/walkthrough-1.6.md (hunk 3 above)
Comment (duglin) [reply docs/walkthrough-1.6.md]: as a newbie I would be confused by the switch in verbs, perhaps do something like:
`Now, let's unbind from (delete) the provisioned instance`

Comment (duglin) [reply docs/walkthrough-1.6.md]: no that's not right either....

Comment (duglin) [reply docs/walkthrough-1.6.md]: skip my comment - we can tweak it in a follow-on PR

**Comment by duglin** [comment]
LGTM

File: docs/walkthrough-1.6.md (hunk 3 above)
Comment (arschles) [reply docs/walkthrough-1.6.md]: @duglin if you'd like to switch the verbs around, I'm happy to do that. It'll require a change in the rest of the doc too, so how about we do that in a follow-up?

**Comment by duglin** [comment]
waiting on CI....

**Comment by arschles** [comment]
thanks @duglin

**Pull request merged**